src/AspNetCoreModuleV2/AspNetCore/InProcessShimStaticHtml.htm

```diff
@@ -1,45 +1,85 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml">
-
-<head>
-    <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
-    <title> IIS 500 Error </title>
-    <style type="text/css"></style>
-</head>
-
-<body>
-    <div id="content">
-        <div class="content-container">
-            <h3> HTTP Error 500.0 - ANCM InProcess Startup Failure </h3>
-        </div>
-        <div class="content-container">
-            <fieldset>
-                <h4> Common causes of this issue: </h4>
-                <ul>
-                    <li> The specified version of Microsoft.NetCore.App or Microsoft.AspNetCore.App was not found. </li>
-                    <li> The in process request handler, Microsoft.AspNetCore.Server.IIS, was not referenced in the application. </li>
-                    <li> ANCM could not find dotnet. </li>
-                </ul>
-            </fieldset>
-        </div>
-        <div class="content-container">
-            <fieldset>
-                <h4> Troubleshooting steps: </h4>
-                <ul>
-                    <li> Check the system event log for error messages </li>
-                    <li> Enable logging the application process' stdout messages </li>
-                    <li> Attach a debugger to the application process and inspect </li>
-                </ul>
-            </fieldset>
-            <fieldset>
-                <h4> For more information visit:
-                    <a href="https://go.microsoft.com/fwlink/?LinkID=808681">
-                        <cite> https://go.microsoft.com/fwlink/?LinkID=808681 </cite>
-                    </a>
-                </h4>
-            </fieldset>
-        </div>
-    </div>
-</body>
+<!DOCTYPE html>
+<html lang="en-US" xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <meta charset="utf-8" />
+        <title> HTTP Error 500.0 - ANCM In-Process Startup Failure </title>
+        <style>
+            body {
+                font-family: 'Segoe UI', Tahoma, Arial, Helvetica, sans-serif;
```

Comment:
I'm not sure how licensing of these fonts work. We'll need to verify if this is kosher

Reply:
Copied from https://github.com/aspnet/Diagnostics/blob/release/2.2/src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/ErrorPage.css

Reply:
Works for me